WordPress/src/test/java/org/wordpress/android/ui/jetpack/BackupStatusServiceTest.kt

```diff
@@ -46,7 +48,7 @@ class BackupDownloadStatusServiceTest {
     @Mock private lateinit var backupDownloadProgressChecker: BackupDownloadProgressChecker
     @Mock private lateinit var site: SiteModel
 
-    private lateinit var backupDownloadStatusService: BackupDownloadStatusService
+    private lateinit var mBackupStatusService: BackupStatusService
```

Comment:
It seems your AS keeps adding "m" prefix even for kotlin files.

Reply:
Yeah, @ParaskP7 called out the same thing above. I'l be changing that.

Reply:
Pushed with c3d265e